### PR TITLE
Assignment-3 done

### PR DESCRIPTION
**Pull request by nikhil1-successive**
Kindly review my Assignment-3

### CODE REVIEW

File: Assignment-3/ques1.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Write a program to perform functionality of a calculator (add,sub,multiply,divide)
+
+function add(n1, n2) {
```
Comment (shahbaz-successive) [inline Assignment-3/ques1.js:3]: Make an actual calculator that takes input and gives output depending upon operator provided.

File: Assignment-3/ques3.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  for (let j = 0; j < i; j++) {
+    string += " ";
+  }
+
+  for (let k = (n - i) * 2 - 1; k > 0; k--) {
+    string += "*";
+  }
+  string += "\n";
+}
+console.log(string);
```
Comment (shahbaz-successive) [inline Assignment-3/ques3.js:27]: Diamond pattern is different from actual required pattern

**Review by shahbaz-successive — CHANGES_REQUESTED** [review]
Please make feedback changes

File: Assignment-3/ques1.js (hunk 1 above)
Comment (nikhil1-successive) [reply Assignment-3/ques1.js]: Done

File: Assignment-3/ques3.js (hunk 2 above)
Comment (nikhil1-successive) [reply Assignment-3/ques3.js]: Done

File: Assignment-3/ques9.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+//  Write a program to print fibonaaci series
+
+function fib(n) {
+  if (n == 0) return;
+  if (n == 1) return 0;
+  if (n == 2) return 1;
```
Comment (shahbaz-successive) [inline Assignment-3/ques9.js:6]: We can write this in single if

**Review by shahbaz-successive — CHANGES_REQUESTED** [review]
Please do feedback changes

File: Assignment-3/ques9.js (hunk 3 above)
Comment (nikhil1-successive) [reply Assignment-3/ques9.js]: Done

**Review by shahbaz-successive — APPROVED** [review]
👍

**Pull request merged**